srtcore/core.cpp

```diff
@@ -9338,6 +9338,8 @@ bool CUDT::overrideSndSeqNo(int32_t seq)
 
 int CUDT::processData(CUnit* in_unit)
 {
+	ScopedLock scopedRecvDataLock(m_RecvDataLock);
```

Comment:
Fixed indent and renamed the variable to be more similar to other names of scoped locks in the code.

```suggestion
    ScopedLock recvdata_scope(m_RecvDataLock);
```